app/src/main/java/com/example/newsapp/ui/home/fragments/EditProfileFragment.kt

```diff
@@ -142,13 +137,13 @@ class EditProfileFragment : Fragment() {
             requireContext().openFileOutput("$fileName.jpg", Context.MODE_PRIVATE)
                 .use { outputStream ->
                     editProfileViewModel.showImage()
-                    if (bitmap.compress(Bitmap.CompressFormat.JPEG, 95, outputStream)) {
+                    if (bitmap.compress(Bitmap.CompressFormat.JPEG, 95, outputStream).not()) {
                         throw IOException(getString(R.string.bitmap_error))
                     }
                 }
             true
         } catch (e: IOException) {
-            Log.e(getString(R.string.error), e.toString())
+            UtilityFunctions.printLogs(getString(R.string.error), e.toString())
```

Comment:
@jnandrawos you should inform the user that something went wrong by showing a message here instead of logging the error